### ISSUE TITLE
Support Node env 

### ISSUE THREAD

**Issue by vipickering** [jamesseanwright/valimate#2]
Is there a way to support environmental variables?

For example this works great:

`"localAppServer": "bin/server.js"`

but something like this does not:

`"localAppServer": "NODE_ENV=test node bin/server.js"`

Perhaps an extra JSON attribute so the path var doesn't have to contain spaces?

**Comment by jamesseanwright** [comment]
Hey there,

Interesting, I hadn't even considered this!

I'm currently wrapping up my entry for js13kGames so I'll fix this next week and release it alongside the resolution for [#1](https://github.com/jamesseanwright/valimate/issues/1).

Cheers,
James

**Comment by vipickering** [comment]
Awesome thanks:-)

I have a test environment mode that turns off authentication (and bypasses certain things) in my Node App, so I can run automated tests like P4lly and this would fit perfectly alongside it.

Good luck in the Competition!

**Comment by jamesseanwright** [comment]
Hey @vipickering,

This is now fixed as of version 2.0.2. The reason for this major release is that the contract for configuring the local app server has changed to accommodate for environment variables:

```
"localAppServer": {
	"entryPoint": "test/testServer.js",

	"env": {
		"TEST": "true"
	}
}
```

Commit [c934567de5416c9daf3f549a71a346cd152c1aca](https://github.com/jamesseanwright/valimate/commit/c934567de5416c9daf3f549a71a346cd152c1aca) details the changes I've made, as well as the functional test to verify that the functionality is working as expected.

I'll close this issue, but please comment and reopen if you're running into trouble.

Cheers,
James
